docs/ext/files.rst

```diff
@@ -0,0 +1,48 @@
+.. _ext-files:
+
+************
+Mopidy-Files
+************
+
+Mopidy-Files is an extension for playing music from your local music archive.
+It is bundled with Mopidy and enabled by default. 
+It allows you to browse through your local file system.
+Only files that are considered playable will be shown
+
+This backend handles URIs starting with ``file:``.
+
+
+Configuration
+=============
+
+See :ref:`config` for general help on configuring Mopidy.
+
+.. literalinclude:: ../../mopidy/files/ext.conf
+    :language: ini
+
+.. confval:: files/enabled
+
+    If the files extension should be enabled or not.
+
+.. confval:: files/media_dir
+
+    A list of directories to be browsable.
+    Each Directory path has to be written in a separate line.
+    Optionally the path can be followed by : and a name that will be shown for that path.
```

Comment:
Is the choice of `:` a wise choice if someone wants to run Mopidy on Windows, given Windows' use of `:` to separate the drive letter and the path? (Running Mopidy on Windows has been done, and is probably still possible for someone dedicated enough.)